src/test/java/org/jabref/logic/layout/format/HTMLCharsTest.java

```diff
@@ -27,16 +27,15 @@ private static Stream<Arguments> provideBasicFormattingData() {
                 Arguments.of("hallo", "hallo"),
                 Arguments.of("Réflexions sur le timing de la quantité", "Réflexions sur le timing de la quantité"),
                 Arguments.of("%%%", "\\%\\%\\%"),
-                Arguments.of("People remember 10%, 20%…Oh Really?", "{{People remember 10\\%, 20\\%…Oh Really?}}"),
```

Comment:
This test should be adapted, because it checks escaped percent signs. Maybe, just add the enclosing double braces to the excpectged output?